### DISCUSSION TITLE
HQ or not HQ

### DISCUSSION THREAD

**Discussion by jackiekazil** [Open-Source-Software-Neighborhood-Watch/town-square#6] [General]
@jspeed-meyers don't hate 😂 
After today's discussion I realized why HQ was not sitting right with me. It works, but it was just off. 

Neighborhood Watches don't have HQs do they? HQ sounds more private sector / corporate to me. 
What is the terminology for a local gathering place in a neighborhood watch?  (Maybe we identify that and we decide that there isn't enough clarity to it, so HQ ends up standing more strong in the end.) 

... and maybe I am just bike shedding.

**Comment by jspeed-meyers** [comment]
@jackiekazil, no hate here! It seemed you didn't like HQ, which is fine! And who's to say what is and isn't bikeshedding. There's a reason I defer to you on making this GitHub group welcoming and appropriate for this community.

Here are some other options:

homebase
central
hub
switchboard
meeting-place
town-square

I'm sure there are many others too.

**Reply by jackiekazil** [reply]
I like town-square or meeting-place... those sound more neighborhood-watch-ish.
Wondering how others feel.

**Reply by jspeed-meyers** [reply]
@tpike3, @gps-iqt-labs, any preferences?

**Reply by gps-iqt-labs** [reply]
Belated +1 for `town-square`

**Reply by tpike3** [reply]
@jspeed-meyers and @jackiekazil I like town-square

**Reply by jspeed-meyers** [reply]
I will change HQ to town-square now.

**Comment by tpike3** [comment]
For next steps... in my mind I am thinking lit review what are all the efforts out there both corporate and not corporate, efforts for recognition and compensation etc.

**Reply by jspeed-meyers** [reply]
That is worthwhile and would help us over the coming months!
